### DISCUSSION TITLE
クライアント側ファイル単位のルーティングについて

### DISCUSSION THREAD

**Discussion by haruki256** [gBenkyous/inTime#82] [General]
クライアント側ファイル単位のルーティングについてファイル単位で自動で設定できたほうが楽だなと感じました。
ただ、next.jsなどはそれが可能なようです。
今後、設定するのとどっちが楽なのかなど見比べるためにも今回はそのままルーティング設定ファイルでの使用を続けようと思っています。

**Comment by doranko** [comment]
どのような機能をもって「ファイル単位で自動で設定」と言っているのかが掴めないので
参考にした記事などあれば紹介いただけますか？

**Comment by haruki256** [comment]
口頭で確認済み
方針下記で問題なし。
今回はそのままルーティング設定ファイルでの使用を続けよう

**Comment by haruki256** [comment]
今後ページ数が増えた場合には、ルーティングのファイルの中身を分割し、コンテンツごとに別のルーティングファイルにまとめる必要性があるかも！
